config/default.json

```diff
@@ -31,5 +30,9 @@
 		"SECRET": "SECRET",
 		"STUDENT_ROOM_CREATION": false
 	},
-	"REQUEST_TIMEOUT": 8000
+	"REQUEST_TIMEOUT": 8000,
+	"TEACHER_STUDENT_VISIBILITY": {
```

Comment:
If you want to use it for development please use development.json. 
The defaults over all should be defined in default.schema.json over the default key. 
The default.json is also set defaults for test and production. 
They is used but let me say: "marked as deprecated in future". 
The only reason why they exist is the also deprecated global.js file and that deep X__Y.default values are not well supported from hpi-common configuration. 

EDIT: Ok i see we have this case with deep defaults.. :D

Reply:
Yes, I added it in default.json because of the issue with deep defaults. :)